ui/StatusQ/src/networkchecker.cpp

```diff
@@ -16,7 +16,7 @@ bool NetworkChecker::isOnline() const
 
 void NetworkChecker::checkNetwork()
 {
-    QNetworkRequest request(QUrl("https://fedoraproject.org/static/hotspot.txt"));
+    QNetworkRequest request(QUrl("http://fedoraproject.org/static/hotspot.txt"));
```

Comment:
```suggestion
    QNetworkRequest request(QUrl(QStringLiteral("http://fedoraproject.org/static/hotspot.txt")));
```

While you're at it :) Btw why the change? Is https not available for you or just slower?

Reply:
Ah, trying to work around the test failure? :thinking:

Reply:
yes. And it doesn't make sense to me to use the https for this purpose.